tests/unit-tests/test-class-sensei-analysis-overview-list-table.php

```diff
@@ -280,4 +287,51 @@ public function testGetLastActivityDateByCourseLessons() {
 			'The last activity date format or timezone is invalid.'
 		);
 	}
+	/**
+	 * Tests that we are getting right data for Completion Rate column.
+	 *
+	 * @covers Sensei_Analysis_Overview_List_Table::get_row_data
+	 * @dataProvider dataFortestCompletionRateForLesson
+	 */
+	public function testCompletionRateForLesson( $enrolled_student_count, $completed_student_count, $expected_output ) {
+		$user_ids                    = $this->factory->user->create_many( $enrolled_student_count );
+		$course_lessons              = $this->factory->get_course_with_lessons(
+			array(
+				'lesson_count' => 1,
+			)
+		);
+		$instance                    = new Sensei_Analysis_Overview_List_Table();
+		$instance->type              = 'lessons';
```

Comment:
You could use the constructor to pass the type. I think it's a nice practice to try to use the constructor as much as possible, it makes refactoring easier.